docs/src/guides/working-with-projects.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Working with Projects
+navTitle: Working with Projects
+description: Working with Hub Projects
+order: 12
+group: 2-concepts
+---
+
+## Working with Projects
+
+### In a Component
+
+If you have a component that you want to fetch and display a project using either
+the project's ID or slug
+
+```html
+<arcgis-hub-project-view identifier="dcdev-smith-st-2024" />
+```
+
+In the component, import the `getProject` function from `@esri/hub-common`, and use that
+in the lifecycle hooks to load the project. Directly importing the helper function ensures that the build will produce the smallest payload as compared to importing `HubProjectStore`.
+
+```ts
+import {getProject} from "@esri/hub-common";
+
+async componentWillLoad() {
+  if (!this.project && this.identifier && this.context) {
+    this.project = await getProject(this.identifier, this.context.requestOptions);
```

Comment:
can this just be `getProject(this.identifier, this.context);`

Reply:
We've made the conscious decision to not require a `context` be passed into Hub.js functions b/c it actually complicates use cases where you want to leverage the functions directly. Rather, `context` exposes getters for the specific request option types.